R/github-pages.R

```diff
@@ -0,0 +1,164 @@
+#' Configure a GitHub Pages site
+#'
+#' Activates or reconfigures a GitHub Pages site for a project hosted on GitHub.
+#' This function anticipates two specific usage modes:
+#' * Publish from the root directory of a `gh-pages` branch, which is assumed to
+#'   be only (or at least primarily) a remote branch.
+#'   - Typically the `gh-pages` branch is managed by an automatic "build and
+#'     deploy" job, such as the one configured by
+#'     [`use_github_action("pkgdown")`][use_github_action()].
+#' * Publish from the `"/docs"` directory of a "regular" branch, probably the
+#'   repo's default branch.
+#'   - The user is assumed to have a plan for how they will manage the content
+#'     below `"/docs"`.
+#'
+
```

Comment:
```suggestion
#'
```

Reply:
I have to leave these blank lanes or else my reflexive use of "reflow comment" in RStudio mangles all bullet lists.